### DISCUSSION TITLE
How to enable fractional scaling?

### DISCUSSION THREAD

**Discussion by AlexValue** [Vanilla-OS/live-iso#364] [Q&A] [answered]
I switched from fedora 39 and there fractional scaling was enabled with the following command:
`gsettings set org.gnome.mutter experimental-features "['scale-monitor-framebuffer']"`

This command does not work in VanillaOS. What should I do to enable fractional display scaling?

**Comment by kbdharun** [comment, marked as answer]
Experimental fractional scaling support is present in Vanilla OS.

In your case, It doesn't work because you might be trying to execute it in the default vso shell which is a mutable container and not in the host.

It will work if you append `host-shell` before the command to execute it on host i.e. `host-shell gsettings set org.gnome.mutter experimental-features "['scale-monitor-framebuffer']"`.

**Reply by AlexValue** [reply]
Thank you! It works

**Reply by pixlxip** [reply]
You can use custom values after turning on fractional scaling from the GNOME Settings and then relog, then edit ~/.config/monitors.xml to change the scales, then relog again. Be sure that if you use a multi-monitor setup, change the x and y of your right/bottom monitor to line up properly with the edge of your left/top monitor and that they do not overlap. If any monitors overlap, GNOME will fall back to the default.
